src/main/java/seedu/duke/CommandList.java

```diff
@@ -24,18 +28,36 @@ public static boolean goalCheck(int userInput, int aiInput) {
         return userInput != aiInput;
     }
 
-    public static void executeShoot(String[] readArgumentTokens) {
+    public static boolean executeShoot(String[] readArgumentTokens) {
         String selectedDirection = readArgumentTokens[0];
         int selectedDirectionIndex = Integer.parseInt(selectedDirection);
-        boolean isScoreGoal = goalCheck(Ai.getAiDirection(), selectedDirectionIndex);
-
-        Formatter.printGoalAfterShot(isScoreGoal);
+        return goalCheck(Ai.getAiDirection(), selectedDirectionIndex);
     }
 
     public static void executePenalty() {
         Penalty.executePenalty();
     }
-    
+
+    public static void executeLoad(String[] readArgumentTokens) {
+        String selectedUser = readArgumentTokens[0];
+        for (int i = 0; i < Cache.players.size(); i++) {
+            if (Cache.players.get(i).name.equals(selectedUser)) {
+                Ui.currentPlayer = Cache.players.get(i);
+                Formatter.printWelcomeUser(selectedUser);
+                System.out.println("Current Difficulty: " + Ui.currentPlayer.difficulty);
+                Ui.roundCount = Ui.currentPlayer.getExp() + 1;
+                Formatter.printGoalBeforeShot(Ui.roundCount);
```

Comment:
I think this part can be outside of the for loop to make the happy path prominent.